projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -29,23 +29,23 @@
   "item.ars_nouveau.lesser_discount_ring": "次等节魔之戒",
   "item.ars_nouveau.greater_discount_ring": "高等节魔之戒",
   "item.ars_nouveau.belt_of_levitation": "浮空腰带",
-  "item.ars_nouveau.belt_of_unstable_gifts": "不稳定礼物腰带",
+  "item.ars_nouveau.belt_of_unstable_gifts": "不定之赐腰带",
   "item.ars_nouveau.mundane_belt": "平凡腰带",
   "item.ars_nouveau.ring_of_potential": "潜能之戒",
   "item.ars_nouveau.jar_of_light": "光之瓶",
   "item.ars_nouveau.bookwyrm_charm": "书龙护符",
   "item.ars_nouveau.starbuncle_charm": "星宝石兽护符",
   "item.ars_nouveau.starbuncle_shards": "星宝石兽碎片",
-  "item.ars_nouveau.creative_spell_book": "创造模式法术书",
+  "item.ars_nouveau.creative_spell_book": "万法通识之书",
   "item.ars_nouveau.dull_trinket": "无趣饰品",
-  "item.ars_nouveau.amulet_of_mana_boost": "魔力提升护身符",
-  "item.ars_nouveau.amulet_of_mana_regen": "魔力再生护身符",
+  "item.ars_nouveau.amulet_of_mana_boost": "魔力提升护符",
+  "item.ars_nouveau.amulet_of_mana_regen": "魔力再生护符",
   "item.ars_nouveau.dominion_wand": "支配之杖",
   "block.ars_nouveau.arcane_ore": "魔源宝石矿石",
   "block.ars_nouveau.arcane_bricks": "奥术砖块",
   "block.ars_nouveau.scribes_table": "抄写台",
   "block.ars_nouveau.light_block": "光",
-  "block.ars_nouveau.mage_block": "法师方块",
+  "block.ars_nouveau.mage_block": "幻影方块",

```

好像dnd对这种法术塑造的东西没有特殊的称呼？我个人偏向是
```suggestion
  "block.ars_nouveau.mage_block": "塑能方块",
```